pkg/utils/konnect/resources/kongdataplaneclientcertificate.go

```diff
@@ -0,0 +1,30 @@
+package resources
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	commonv1alpha1 "github.com/kong/kubernetes-configuration/api/common/v1alpha1"
+	configurationv1alpha1 "github.com/kong/kubernetes-configuration/api/configuration/v1alpha1"
+)
+
+// GenerateKongDataPlaneClientCertificate generates a KongDataPlaneClientCertificate object .
```

Comment:
```suggestion
// GenerateKongDataPlaneClientCertificate generates a KongDataPlaneClientCertificate object
// using the provided controlPlaneRef.
```

- can we also comment on the cert param here, mentioning that this is the actual cert (its contents)?